packages/suite-base/src/providers/LayoutManagerProvider.test.tsx

```diff
@@ -61,212 +52,6 @@ describe("LayoutManagerProvider", () => {
     });
   });
 
-  it("should not call getLayouts or saveNewLayout if layout loaders is undefined or an empty array", async () => {
```

Comment:
There was no way of keeping those tests?

Reply:
As I moved the implementation from LayoutManagerProvider to CurrentLayoutProvider, it doesn't make sense to keep these tests here. But instead of implement directly on the CurrentLayoutProvider, I extract the logic to another file, and these tests were replicated there. You can check the same tests here:
`packages/suite-base/src/providers/CurrentLayoutProvider/loadDefaultLayouts.test.ts`